docs/iris/src/whatsnew/latest.rst

```diff
@@ -13,193 +13,255 @@ This document explains the changes made to Iris for this release
 
-* The :mod:`iris.experimental.concatenate` module has now been removed. In
+* `Bill Little`_ removed the :mod:`iris.experimental.concatenate` module. In
   ``v1.6.0`` the experimental ``concatenate`` functionality was moved to the
   :meth:`iris.cube.CubeList.concatenate` method.  Since then, calling the
   :func:`iris.experimental.concatenate.concatenate` function raised an
-  exception.
+  exception. (:pull:`3523`)
 
-* When loading data from NetCDF-CF files, where a variable has no ``units``
-  property, the corresponding Iris object will have ``units='unknown'``.
-  Prior to Iris ``3.0.0``, these cases defaulted to ``units='1'``.
-  See :pull:`3795`.
+* `stephenworsley`_ changed Iris objects loaded from NetCDF-CF files to have
+  ``units='unknown'`` where the corresponding NetCDF variable has no ``units``
+  property. Previously these cases defaulted to ``units='1'``. (:pull:`3795`)
 
 
-Internal
-========
+Deprecations
+============
 
-* Changed the numerical values in tests involving the Robinson projection due
-  to improvements made in `Proj <https://github.com/OSGeo/PROJ>`_ (see
-  `proj#1292 <https://github.com/OSGeo/PROJ/pull/1292>`_ and
-  `proj#2151 <https://github.com/OSGeo/PROJ/pull/2151>`_).
+* `stephenworsley`_ removed the deprecated :class:`iris.Future` flags
+  ``cell_date_time_objects``, ``netcdf_promote``, ``netcdf_no_unlimited`` and
+  ``clip_latitudes``. (:pull:`3459`)
 
-* Change tests to account for more detailed descriptions of projections in
-  `GDAL <https://github.com/OSGeo/gdal>`_ 
-  (`see GDAL#1185 <https://github.com/OSGeo/gdal/pull/1185>`_).
+* `stephenworsley`_ changed :attr:`iris.fileformats.pp.PPField.lbproc` is be an
+  ``int``. The deprecated attributes ``flag1``, ``flag2`` etc. have been
+  removed from it. (:pull:`3461`).
 
-* Change tests to account for `GDAL <https://github.com/OSGeo/gdal>`_ now
-  saving fill values for data without masked points.
 
-* Changed every graphics test that includes `Cartopy's coastlines
-  <https://scitools.org.uk/cartopy/docs/latest/matplotlib/
-  geoaxes.html?highlight=coastlines#cartopy.mpl.geoaxes.GeoAxes.coastlines>`_
-  to account for new adaptive coastline scaling (`see cartopy#1105
-  <https://github.com/SciTools/cartopy/pull/1105>`_).
+Dependencies
+============
 
-* Changed graphics tests to account for some new default grid-line spacing in
-  `Cartopy <https://github.com/SciTools/cartopy>`_ (`part of cartopy#1117
-  <https://github.com/SciTools/cartopy/pull/1117>`_).
 
-* Additional acceptable graphics test targets to account for very minor changes
-  in `Matplotlib`_ version 3.3 (colormaps, fonts and axes borders).
+* `stephenworsley`_, `Martin Yeo`_ and `Bill Little`_ removed ``Python2``
+  support, modernising the codebase by switching to exclusive ``Python3``
+  support. (:pull:`3513`)
 
+* `Bill Little`_ improved the developer set up process. Configuring Iris and
+  :ref:`installing_from_source` as a developer, with all the required package
```

Comment:
Comma should probably be removed

Reply:
d1038c403d23de9e461e3fe5e3a94cb7def73778